worldedit-core/src/main/java/com/sk89q/worldedit/command/BrushCommands.java

```diff
@@ -320,6 +323,37 @@ public void butcherBrush(Player player, LocalSession session,
         player.printInfo(TranslatableComponent.of("worldedit.brush.butcher.equip", TextComponent.of((int) radius)));
     }
 
+    @Command(
+        name = "image",
+        desc = "Image brush, uses images as input"
+    )
+    @CommandPermissions("worldedit.brush.image")
+    public void imageBrush(Player player, LocalSession session,
```

Comment:
It would be preferred to leave these methods as package-private, I know most of the others are `public` (I forgot to change them at the time) but new ones should be consistent.